django_freeradius/base/admin.py

```diff
@@ -150,3 +150,22 @@ class AbstractRadiusPostAuthAdmin(BasePostAuth):
     list_display = ['username', 'reply', 'date']
     list_filter = ['date', 'reply']
     search_fields = ['username', 'reply']
+
+
+class AbstractRadiusBatchAdmin(TimeStampedEditableAdmin):
+    readonly_fields = ('users',)
+
+    def save_model(self, request, obj, form, change):
```

Comment:
can we do this within the model in some way as well? That way we can reuse the logic from an API too in the future.
If we wanted to implement an API for this operation we'd have to duplicate this logic.

Reply:
Refactored some part of this function.